concordia/views.py

```diff
@@ -1394,6 +1394,7 @@ def serialize_object(self, obj):
                 "url": campaign.get_absolute_url(),
             },
             "latest_transcription": obj.latest_transcription,
+            "last_updated": int(time.time()),
```

Comment:
`last_updated` could be interpreted as the record's modification time. I'm thinking that we could move this out of the per-object serialization and just include it a single time in the response, probably by adding it to the API view base class's `serialize_context`.